docs/apps/seqtk.md

```diff
@@ -0,0 +1,63 @@
+
+# Seqtk
+
+## Description
+
+Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
+
+
+llumina BaseSpace command line client,`bs`, can be used to retrievie data from the Illumina BaseSpace environment to Puhti.
+
+## Available
+
+*    bs version 1.0.0 is available in Puhti
+
+## Usage
+
+
+Seqtk command syntax is:
+```text
+seqtk <command> <arguments>
+```
```

Comment:
The `seqtk` command is not in path. Does the user need to load/set/install something for these instructions to work?

Reply:
korjattu